fesvr/htif.cc

```diff
@@ -246,8 +246,11 @@ void htif_t::parse_arguments(int argc, char ** argv)
           optarg = optarg + 8;
         }
         else {
-          optind--;
-          goto done_processing;
+          if(arg.find("+") != 0) {
```

Comment:
Doesn't this say, "if the argument contains a ```+``` anywhere besides the first character, stop parsing arguments"?  Should it be ```if (arg.find("+") == std::string::npos)```?

Reply:
I think this is okay. `find` should return something non-zero (or `npos` which is a big number) if it finds something that doesn't start with a `+`. Looking for `npos` would be too restrictive as there could be a binary with a `+` in the middle of it. It's a stretch, but possible.